### PR TITLE
Test/add tests to slider image component

### PR DESCRIPTION
**Pull request by Isaac-alencar**
# Description
- This Pull Request add tests to `SliderImage`. But I also made some minor improvements on component itself
- improve typing avoiding components with "any" type
- Improvements on storybook adding more assets and a video asset to see how it works with multiple assets

## Does this pull request close any open issues?
- closes #257

### CODE REVIEW

File: src/components/moleculars/sliders/SliderImage/index.stories.tsx [hunk 1]
```diff
@@ -16,5 +16,7 @@ Default.args = {
   sliderImages: [
     { imageUrl: "https://picsum.photos/600/600" },
     { imageUrl: "https://picsum.photos/600/600?random=2" },
+    { imageUrl: "https://picsum.photos/600/600?random=3" },
+    { imageUrl: "https://img-9gag-fun.9cache.com/photo/a81OBv1_460svvp9.webm" },
```
Comment (leodhb) [inline src/components/moleculars/sliders/SliderImage/index.stories.tsx:20]: Can we use another picsum image here too?

File: src/components/moleculars/sliders/SliderImage/index.test.tsx [hunk 2]
```diff
@@ -1,12 +1,149 @@
...
+    });
+  });
+
+  describe("when a video link is passed as prop ", () => {
+    it("should render video instead an image", () => {
+      const sliderProps = {
+        sliderImages: [
+          {
+            imageUrl:
+              "https://img-9gag-fun.9cache.com/photo/a81OBv1_460svvp9.webm",
```
Comment (leodhb) [inline src/components/moleculars/sliders/SliderImage/index.test.tsx:139]: Ditto

**Review by leodhb — CHANGES_REQUESTED** [review]

File: src/components/moleculars/sliders/SliderImage/index.test.tsx (hunk 2 above)
Comment (leodhb) [inline src/components/moleculars/sliders/SliderImage/index.test.tsx:139]: Ditto

**Review by leodhb — CHANGES_REQUESTED** [review]

**Comment by Isaac-alencar** [comment]
Apologies for delay by EOW I will deliver all changes :)

**Pull request closed without merging**